tests/services/getQuestParticipant.test.js

```diff
@@ -0,0 +1,68 @@
+import { getQuestsParticipation } from "@services/apiService";
+
+const API_URL = process.env.NEXT_PUBLIC_API_LINK;
+
+global.fetch = jest.fn();
+
+describe("getQuestsParticipation", () => {
+  beforeEach(() => {
+    fetch.mockClear();
+  });
+
+  it("should fetch and return data for a valid id", async () => {
+    const mockData = {
```

Comment:
i would suggest using the api once to see what is the ideal response you get from the api for this and also add that as a return type in the apiService.ts